builder_automate.md

```diff
@@ -0,0 +1,389 @@
+# Install Chef Habitat Builder On-Premises
+
+The Habitat 0.85.0 release simplifies custom certificate management. Habitat now looks for custom certificates in its `~/.hab/cache/ssl` directory (or `/hab/cache/ssl` if running as root). You can copy over Multiple certificates--for example, a self-signed certificate and a custom certificate authority certificate--to the cache directory and they are automatically available for use by the Habitat client.
+
+## System Requirements
+
+The following are minimum requirements for the Chef Habitat On-Prem Builder:
+
+* Linux OS system with 64-bit architecture with kernel 2.6.32 or later
+* `systemd` process manager
+* CPU / RAM for trial deployments: minimum 2 CPU/4 GB RAM (corresponding to AWS T3.medium or better) or better
+* CPU / RAM for production deployments: minimum 16 CPU/32 GB RAM (corresponding to AWS M4.4xlarge) or better
+* Disk space for trial deployments: Minimum 2GB free disk space the baseline installation with only the packages required to run the Builder Services
+* Disk space for production deployments: Minimum 5GB+ of disk space for the full installation including the latest versions of core packages
+* Services should be deployed single-node - scale out is not yet supported
+* Outbound network (HTTPS) connectivity to WAN is required for the initial installation
+* Inbound network connectivity from LAN (HTTP/HTTPS) is required for internal clients to access the Builder
+* SSL Certificate
+* Oauth 2
+* Port 80 or 443
+
+## Authentication
+
```

Comment:
Since this is for Automate specifically we can remove the list of other OAuth providers